ShitFood.Api/GetShit.cs

```diff
@@ -9,33 +9,76 @@
 using Newtonsoft.Json;
 using System.Net.Http;
 using ShitFood.Api.FoodHygieneApi;
+using GoogleApi;
+using GoogleApi.Entities.Places.Search.Find.Request;
+using GoogleApi.Entities.Places.Search.Find.Response;
+using System.Linq;
+using System.Collections.Generic;
```

Comment:
For your own sanity, do you want to group/sort these?

Reply:
Unlike JS stuff, Visual Studio does a pretty good job at looking after imports, so keeping them in alphabetical order isn't necessary imo.

Reply:
That's fine but I meant like grouping `System` and `GoogleApi`, but it doesn't matter too much B-)